Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.5.5-jessie
+
+# TODO: move in a base image
+RUN useradd -ms /bin/bash -G www-data elife && \
+    chown elife:elife /srv && \
+    mkdir /srv/bin && \
+    chown elife:elife /srv/bin && \
+    mkdir -p /var/www && \
+    chown www-data:www-data /var/www
+
+RUN pip install -U virtualenv
+RUN pip install -U setuptools
+# END base image
+
+USER elife
+ENV PROJECT_FOLDER=/srv/profiles
+ENV LANG=en_US.UTF-8
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR /srv/profiles
+COPY --chown=elife:elife install.sh requirements.txt /srv/profiles/
+RUN PROFILES_SKIP_DB=1 /bin/bash install.sh
+
+## yes this is how you copy directories
+COPY --chown=elife:elife manage.py /srv/profiles/
+COPY --chown=elife:elife migrations /srv/profiles/
+COPY --chown=elife:elife profiles/ /srv/profiles/profiles
+
+RUN mkdir -p var/logs
+USER root
+RUN chown www-data:www-data var/logs
```

Comment:
Could be combined with the `mkdir`?

Reply:
the `mkdir` is executed as `elife` so that it leaves `var/` owned as `elife:elife`, while `var/logs/` becomes owned by `www-data:www-data`; change of ownership requires root

Reply:
`var` is in the repo but doesn't have a `COPY`, for clarity either just `mkdir -p var` as `elife` or add one (`COPY`) above and move the `mkdir` here. That would make it a bit clearer.

Reply:
`COPY` is dangerous since it may pick up local log files and bake them into the image, whereas the folder should be empty by default; so I'll move the `var/` creation up